src/firefly/java/edu/caltech/ipac/firefly/server/servlets/ServerStatus.java

```diff
@@ -119,11 +105,19 @@ private static void displayCacheInfo(PrintWriter writer, CacheManager cm) {
     private static void showDatabaseStatus(PrintWriter writer) {
         writer.println("DATABASE INFORMATION");
         writer.println("--------------------");
-        writer.println("Open: " + BaseDbAdapter.getDbInstances().size());
-        writer.println("Details: idle time is in (mm:ss)");
-        Collections.unmodifiableCollection(BaseDbAdapter.getDbInstances().values()).stream()
+        writer.println(String.format("Open: %d    CHECK_INTVL: %ds    MAX_ROWS: %d    MAX_IDLE: %dm",
+                        DbAdapter.getAdapter().getDbInstances().size(), DbAdapter.CLEANUP_INTVL/1000, DbAdapter.MAX_MEMORY_ROWS, DbAdapter.MAX_IDLE_TIME/1000/60));
+        writer.println("Idled   Age     Tables  Rows      File Path");
+        writer.println("------  ------  ------  --------  ---------");
```

Comment:
Is it possible to do row and columns? They way is would be easier to estimate table size